### DISCUSSION TITLE
PatchTool 6.08 4/7 - fixes for "Official Translation"

### DISCUSSION THREAD

**Discussion by GpointChen** [GpointChen/FFXIVChnTextPatch-GP#121] [Announcements]
- 修正 #117 #118 #119 。
  - 開心胡蘿蔔改為輸入「carrot of happiness」。

- FFXIVChnTextPatch-0407.rar：基本包。一般使用者下載這個即可。

其他可選：

- [第三方4K字體包](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.53/font-4k.rar)：部分解析度設定下會出現字體錯位等情形，請自行照需求使用。解壓後改名為font放到resource資料夾下（取代原資料夾）。
- rawexd-zhcn-0407.rar：簡體中文CSV。解壓後改名為rawexd放到resource資料夾下（取代原資料夾）。
- [FFXIV-jar-0211.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.56/FFXIV-jar-0211.rar)：如果你有遇到JNI error，請參考 #90 放到與exe同位置使用。
- [backup-0320.rar](https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/download/v2.55/backup-0320.rar)：乾淨的0x0000檔案。如果漢化出現問題，可以將這些檔案放到backup後按還原，然後再次嘗試漢化。

有問題歡迎至Issues或Discussions回報。

<hr /><em>This discussion was created from the release <a href='https://github.com/GpointChen/FFXIVChnTextPatch-GP/releases/tag/v2.57'>PatchTool 6.08 4/7 - fixes for "Official Translation"</a>.</em>

**Comment by QPyou** [comment]
想請問安裝漢化後開遊戲會出現dx錯誤和一個KERNEL32.DLL跟ntdll.dll的錯誤怎麼辦呢?

**Reply by GpointChen** [reply]
6.1改版所致，等我更新吧。

**Comment by arwuu** [comment]
您好，想詢問一下再安裝漢化(替換文本那邊)會跳出漢化錯誤 (程式錯誤) 請問一下該怎麼辦呢?
也有嘗試還原過了QQ

**Reply by GpointChen** [reply]
6.1改版所致，等我更新吧。
